docs/schemas/2024-04/donor-details.json

```diff
@@ -0,0 +1,263 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://data-dictionary.opg.service.justice.gov.uk/schema/lpa/2024-04/lpa-donor-details.json",
```

Comment:
should this match `/donor-details.json`?

Reply:
Yep yep yep, thank you fixed